File: Source/WickedSick.Server.XamlParser/Elements/Media/Animation/ColorAnimationUsingKeyFrames.cs

namespace WickedSick.Server.XamlParser.Elements.Media.Animation
{
    [Element("Fayde.Media.Animation")]
    public class ColorAnimationUsingKeyFrames : Timeline
    {
        public static readonly PropertyDescription KeyFramesProperty = PropertyDescription.Register("KeyFrames", typeof(DependencyObjectCollection<ColorKeyFrame>), typeof(ColorAnimationUsingKeyFrames), true);
    }
}

File: Source/Fayde.Client/Media/Animation/ColorAnimationUsingKeyFrames.ts
/// <reference path="AnimationUsingKeyFrames.ts" />

module Fayde.Media.Animation {
    export class ColorAnimationUsingKeyFrames extends AnimationUsingKeyFrames {
        static Annotations = { ContentProperty: AnimationUsingKeyFrames.KeyFramesProperty };
    }
    Fayde.RegisterType(ColorAnimationUsingKeyFrames, "Fayde.Media.Animation", Fayde.XMLNS);
}